packages/client/src/context.tsx

```diff
@@ -1,14 +1,11 @@
+import { AnswerDict } from "@vergunningcheck/imtr_client";
```

Comment:
Without thinking about it, the term `AnswerDict` doesn't mean anything to me. Can you rename to something more appealing?

Reply:
Hmm maybe my python background 😊
Do you think AnswersObject or AnswerCollection is better?

The difference is `Answer[]` vs `{[id: string]: Answer}`.

Reply:
Why not `AnswersProp` or just `Answers`? Otherwise `AnswersObject` is fine.